server/src/main/java/io/crate/metadata/functions/Signature.java

```diff
@@ -418,6 +453,8 @@ public void writeTo(StreamOutput out) throws IOException {
         }
         TypeSignature.toStream(returnType, out);
         out.writeVInt(EnumSets.packToInt(features));
+        Integer nullabilityOrdinal = nullability == null ? null : nullability.ordinal();
```

Comment:
same here about version check for the streaming,
Please make sure it's properly tested!